Aspire Budgeting/Facilities/AspireOSLogs.swift

```diff
@@ -30,4 +30,6 @@ extension OSLog {
   static let stateManager = OSLog(subsystem: OSLog.subsystem, category: "StateManager")
 
   static let dashboardMetadata = OSLog(subsystem: OSLog.subsystem, category: "DashboardMetadata")
+  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/fd43c30d9ee7d075625997a607bdb1dd002cf1a8/Aspire Budgeting/Facilities/AspireOSLogs.swift#L33"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>AspireOSLogs.swift:33</code></td>
    </tr>
  </tbody>
</table>